### PR TITLE
Fix RMS_eps

### PR DESCRIPTION
**Pull request by gregor-soniox**
This ensures that logits from lit-llama model are "the same" as logits of meta/llama repo. Also referencing https://discord.com/channels/1077906959069626439/1110860278792466512

### CODE REVIEW

**Comment by awaelchli** [comment]
Hi @gregor-soniox, thanks for the PR. FYI In our tests we already set this to 1e-6, where we check for parity with the original model: https://github.com/Lightning-AI/lit-llama/blob/main/tests/test_rmsnorm.py. Since this is the new default, could you remove the explicit setting from the test code? This way our tests will rely on the default values in the model, and make your change "future proof". How does that sound?

**Comment by gregor-soniox** [comment]
I think we still want to set the "default" llama to 1e-6, but yeah removing from lit llama constructor sounds good, will fix.

**Review by carmocca — APPROVED** [review]
To give context, the original LLaMA release set `1e-5` in the source code but the checkpoint release (and HF checkpoint copies) use `1e-6`.

Since we download the original model implementation for comparison during testing, we made this mistake.

So https://github.com/Lightning-AI/lit-llama/blob/main/tests/test_model.py#L53 should be updated too

**Comment by rasbt** [comment]
Ah nice! Maybe that will result in better normalization when using regular float16 training (versus bfloat16)

**Comment by carmocca** [comment]
@gregor-soniox Did you intend to close this PR? Would you like that I re-open it?
